share/commands/agent-check

```diff
@@ -200,30 +200,16 @@ empty() {
 
 check_and_fix_inputs() {
   # if file is absent or empty there have been a problem with update
-  if empty "${CFE_DIR}/inputs/common/1.0/update.cf" || empty "${CFE_DIR}/inputs/failsafe.cf" || empty "${CFE_DIR}/inputs/promises.cf" || ${CFE_BIN_DIR}/cf-promises -f failsafe.cf | grep error > /dev/null
+  if empty "${CFE_DIR}/inputs/common/1.0/update.cf" || empty "${CFE_DIR}/inputs/failsafe.cf" || empty "${CFE_DIR}/inputs/promises.cf" || ${CFE_BIN_DIR}/cf-promises -f failsafe.cf | grep error > /dev/null || ${CFE_BIN_DIR}/cf-promises | grep error > /dev/null
```

Comment:
we had the goal to change the frequency of cf-promises, to splay it and run it less often - with keeping the check on empty files at each run
this would prevent to have a difference on ressource intensive check and lightweight ones

Reply:
This can be done when we implement the different frequency of cf-promises